api/HTMLElement.json

```diff
@@ -1144,7 +1144,8 @@
               "version_added": "45"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "8",
+              "notes": "Internet Explorer 5.5 initially implemented this feature on the <code>Element</code> API rather than the <code>HTMLElement</code> API."
```

Comment:
@Elchi3 do you have an opinion on how to represent this data? It's rather historic, so maybe there's no harm leaving this in a note. But I could also see an argument for having it be `"version_added": "5.5"`, or to pretend this didn't exist before 8. 🤷‍♂️

Reply:
tl;dr: Make it version_added: 5.5

I believe in the historic context, HTMLElement and Element are very similar. Nowadays, the two classes might differentiate more (in the light of SVG, MathML, etc). So, I think `version_added: 5.5` is the least strange to people given what the issue reporter described.
APIs move around sometimes and BCD can't represent that nicely right now, we have an issue filed on that, though. I think this is also connected to how MDN documents APIs (messy right now), so I only see a nice solution to this when we researched better API doc structures that take into account topics like inheritance etc. So, if MDN is clear, BCD can then try document how browsers variate in terms of API surfaces.

Reply:
Unfortunately this conflicts with the consistency linter, hence why I added it as a note instead of simply setting the version_added.  Should we bump the API version itself to 5.5 as well, in that case?

Reply:
Yes, my feeling is to bump that to 5.5 as well. I haven't tested if this class is in 5.5, but I'd expect so.

Reply:
Actually, the HTMLElement API wasn't supported until IE 8 for some strange reason -- I found this out when working on the API KR.  But I'd be happy to bump the version anyways!

Reply:
I'd say bump anyway.